mne/io/fiff/tests/test_raw_fiff.py

```diff
@@ -771,6 +771,32 @@ def test_filter():
     assert_array_almost_equal(data, data_notch, sig_dec_notch_fit)
 
 
+def test_filter_picks():
+    """Test filtering default channel picks"""
+    ch_types = ['mag', 'grad', 'eeg', 'seeg', 'misc', 'stim']
+    info = create_info(ch_names=ch_types, ch_types=ch_types, sfreq=256)
+    raw = RawArray(data=np.zeros((len(ch_types), 1000)), info=info)
+
+    # -- Deal with meg mag grad exception
+    ch_types = ('misc', 'stim', 'meg', 'eeg', 'seeg')
+
+    # -- Filter data channels
+    for ch_type in ('mag', 'grad', 'eeg', 'seeg'):
+        picks = {ch: ch == ch_type for ch in ch_types}
+        picks['meg'] = ch_type if ch_type in ('mag', 'grad') else False
+        raw_ = raw.pick_types(copy=True, **picks)
+        with warnings.catch_warnings(record=True) as w:
```

Comment:
make a comment here saying what the warning is, it's not obvious from the code or the assertion check